publication/src/main/scala/hmda/publication/reports/aggregate/A8X.scala

```diff
@@ -0,0 +1,348 @@
+    (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 3)
+  }
+}
+
+object A84 extends A8X {
+  val reportId = "A84"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 2)
+  }
+}
+
+object A85 extends A8X {
+  val reportId = "A85"
+  def filters(lar: LoanApplicationRegister): Boolean = lar.loan.propertyType == 3
+}
+
+object A86 extends A8X {
+  val reportId = "A86"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    lar.loan.occupancy == 2 &&
+      (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 1 || lar.loan.purpose == 2 || lar.loan.purpose == 3)
+  }
+}
+
+object A87 extends A8X {
+  val reportId = "A87"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    lar.loan.propertyType == 2 &&
+      (lar.loan.purpose == 1 || lar.loan.purpose == 2 || lar.loan.purpose == 3)
+  }
+}
+
+object N81 extends A8X {
+  val reportId = "N81"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    (lar.loan.loanType == 2 || lar.loan.loanType == 3 || lar.loan.loanType == 4) &&
+      (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 1)
+  }
+}
+
+object N82 extends A8X {
+  val reportId = "N82"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    lar.loan.loanType == 1 &&
+      (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 1)
+  }
+}
+
+object N83 extends A8X {
+  val reportId = "N83"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 3)
+  }
+}
+
+object N84 extends A8X {
+  val reportId = "N84"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 2)
+  }
+}
+
+object N85 extends A8X {
+  val reportId = "N85"
+  def filters(lar: LoanApplicationRegister): Boolean = lar.loan.propertyType == 3
+}
+
+object N86 extends A8X {
+  val reportId = "N86"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    lar.loan.occupancy == 2 &&
+      (lar.loan.propertyType == 1 || lar.loan.propertyType == 2) &&
+      (lar.loan.purpose == 1 || lar.loan.purpose == 2 || lar.loan.purpose == 3)
+  }
+}
+
+object N87 extends A8X {
+  val reportId = "N87"
+  def filters(lar: LoanApplicationRegister): Boolean = {
+    lar.loan.propertyType == 2 &&
+      (lar.loan.purpose == 1 || lar.loan.purpose == 2 || lar.loan.purpose == 3)
+  }
+}
+
+trait A8X extends AggregateReport {
+  val reportId: String
+  def filters(lar: LoanApplicationRegister): Boolean
+
+  val dispositions = List(DebtToIncomeRatio, EmploymentHistory, CreditHistory,
+    Collateral, InsufficientCash, UnverifiableInformation, CreditAppIncomplete,
+    MortgageInsuranceDenied, OtherDenialReason)
```

Comment:
This report should also have the `TotalDenied` disposition. That disposition is also missing from the `D8X` series. Could you update both `A8X` and `D8X` to include it? (or create a ticket for adding it to `D8X`, but doing it in this PR seems easier)